src/perf/README.md

```diff
@@ -0,0 +1,45 @@
+# Running tests in Gatling
+
+1. Download Gatling: http://gatling.io/#/download
+2. Point your gatling.conf located in {GATLING_HOME}/conf to use the simulations in {RAWLS_HOME}/src/perf
+	Locations are configurable under "directory"
+
+	It should look like:
+
+	    directory {
+          #data = user-files/data 
+          #bodies = user-files/bodies
+          simulations = {your_rawls_home}/src/perf/simulations
+          #reportsOnly = ""
+          #binaries = ""
+          #results = results
+        }
+
+    Make sure those lines are uncommented!
+
+3. Create a plaintext config.txt file in your {GATLING_HOME}/user-files which is of the following form:
+
+    	{YOUR_ACCESS_TOKEN}
+    	{NUM_USERS}
+
+	Example:
+
+    	ya29.ZZZZZZ_zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
+        100
+
+4. Run ./gatling.sh in located in {GATLING_HOME}/bin
+5. A list of valid simulations will pop up. Select one that excites you.
```

Comment:
Add that it's not necessary to name the simulation ID, and also describe what naming it will do